pom.xml

```diff
@@ -942,6 +944,16 @@
                 <artifactId>commons-text</artifactId>
                 <version>${version.lib.commons-text}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>${version.lib.maven-shade-plugin}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse</groupId>
+                <artifactId>yasson</artifactId>
```

Comment:
Use Parsson if you must use JSON-P, Yasson is a JSON-B implementation.

Reply:
thx, fixed

Reply:
The old JSON-P is already in-use in build-tools, this would introduce the new one without upgrading existing code.